### DISCUSSION TITLE
Indian companies financials

### DISCUSSION THREAD

**Discussion by poonkundran-k** [JerBouma/FinanceDatabase#43] [Q&A] [answered]
I am not able to access Indian company financials. 

How to access the same?

**Comment by JerBouma** [comment]
There is plenty of data within:

```
equities = fd.Equities()
equities.search(country='India')
equities.search(market='India')
```

For updated financial statements, my recommendation would be to search for symbols listed on American exchanges and use the FinanceToolkit with those symbols. See: https://github.com/JerBouma/FinanceToolkit

E.g. for Tata Motors:

```
from financetoolkit import Toolkit

tata_motors = Toolkit('TTM', api_key=FINANCIALMODELINGPREP_KEY)

tata_motors.get_balance_sheet_statement()
```

Which returns:

|                                          |          2018 |          2019 |          2020 |          2021 |          2022 |
|:-----------------------------------------|--------------:|--------------:|--------------:|--------------:|--------------:|
| Cash and Cash Equivalents                |  147167500000 |  215598000000 |  184678000000 |  317000100000 |  381590100000 |
| Short Term Investments                   |  345226700000 |  195125400000 |  256910100000 |  333971800000 |  247468000000 |
| Cash and Short Term Investments          |  492394200000 |  410723400000 |  441588100000 |  650971900000 |  629058100000 |
| Accounts Receivable                      |  345900700000 |  363928400000 |  303683800000 |  363749000000 |  395011500000 |
| Inventory                                |  424296200000 |  390015900000 |  374527800000 |  360865000000 |  352446000000 |
| Other Current Assets                     |  100057200000 |   63607500000 |   76036000000 |   93273600000 |   93027800000 |
| Total Current Assets                     | 1362648300000 | 1228275200000 | 1195835700000 | 1468859500000 | 1469543400000 |
| Property, Plant and Equipment            |  850444100000 |  762303300000 |  878462000000 |  887947700000 |  855994100000 |
| Goodwill                                 |    1164500000 |    7478700000 |    7770600000 |    8037200000 |    8071700000 |
| Intangible Assets                        |  685167900000 |  580555900000 |  664544500000 |  614483300000 |  546811700000 |
| Long Term Investments                    |   65831600000 |   70468300000 |   56786900000 |   63095500000 |   74766600000 |
| Tax Assets                               |   41064600000 |   51511100000 |   54578600000 |   45203500000 |   38708700000 |
| Other Fixed Assets                       |  229616200000 |  286527400000 |  282823800000 |  255167700000 |  229706500000 |
| Fixed Assets                             | 1873288900000 | 1758844700000 | 1944966400000 | 1873934900000 | 1754059300000 |
| Other Assets                             |             0 |             0 |             0 |             0 |             0 |
| Total Assets                             | 3235937200000 | 2987119900000 | 3140802100000 | 3342794400000 | 3223602700000 |
| Accounts Payable                         |  804601600000 |  759681700000 |  705671900000 |  739739600000 |  649452800000 |
| Short Term Debt                          |  277509700000 |  352016700000 |  363090800000 |  436057400000 |  427274100000 |
| Tax Payables                             |   47359300000 |   49315400000 |   46966000000 |   48161300000 |   53976000000 |
| Deferred Revenue                         |   37562400000 |   45773800000 |   42566300000 |   57359300000 |   65586500000 |
| Other Current Liabilities                |  260749700000 |  240963000000 |  239103500000 |  288911200000 |  355143300000 |
| Total Current Liabilities                | 1427782700000 | 1447750600000 | 1397398500000 | 1570228800000 | 1497456700000 |
| Long Term Debt                           |  611886900000 |  709628700000 |  884677500000 |  985140300000 | 1049607900000 |
| Deferred Revenue Non Current             |   41116200000 |   46730900000 |   50157700000 |   48477000000 |   45359000000 |
| Deferred Tax Liabilities                 |   61257800000 |   14910400000 |   19418700000 |   15558900000 |   15584400000 |
| Other Non Current Liabilities            |  179946300000 |  210031900000 |  191115000000 |  204253600000 |  175030100000 |
| Total Non Current Liabilities            |  894207200000 |  981301900000 | 1145368900000 | 1253429800000 | 1285581400000 |
| Other Liabilities                        |             0 |             0 |             0 |             0 |             0 |
| Capital Lease Obligations                |     689800000 |    1734700000 |   59771200000 |   62260600000 |   67719800000 |
| Total Liabilities                        | 2321989900000 | 2429052500000 | 2542767400000 | 2823658600000 | 2783038100000 |
| Preferred Stock                          |             0 |             0 |             0 |             0 |             0 |
| Common Stock                             |    6792200000 |    6792200000 |    7195400000 |    7658100000 |    7658800000 |
| Retained Earnings                        |             0 |             0 |             0 |             0 |             0 |
| Accumulated Other Comprehensive Income   |  634764400000 |  317926700000 |  276352500000 |   74667600000 |  134538800000 |
| Other Total Shareholder Equity           |  267033200000 |  228019800000 |  306253300000 |  420977200000 |  255558400000 |
| Total Shareholder Equity                 |  908589800000 |  552738700000 |  589801200000 |  503302900000 |  397756000000 |
| Total Equity                             |  908589800000 |  552738700000 |  589801200000 |  503302900000 |  397756000000 |
| Total Liabilities and Shareholder Equity | 3230579700000 | 2981791200000 | 3132568600000 | 3326961500000 | 3180794100000 |
| Minority Interest                        |             0 |             0 |             0 |             0 |             0 |
| Total Liabilities and Equity             | 3230579700000 | 2981791200000 | 3132568600000 | 3326961500000 | 3180794100000 |
| Total Investments                        |  411058300000 |  265593700000 |  313697000000 |  397067300000 |  322234600000 |
| Total Debt                               |  889396600000 | 1061645400000 | 1247768300000 | 1421197700000 | 1476882000000 |
| Net Debt                                 |  742229100000 |  846047400000 | 1063090300000 | 1104197600000 | 1095291900000 |

**Reply by poonkundran-k** [reply]
Thank you for you reply. It seems that we can get the data only for ADR.  If I am using WIPRO.BO in the above code, I am getting the below error.
**KeyError: "['symbol'] not found in axis"**

**Reply by JerBouma** [reply, marked as answer]
This is correct, the FinanceToolkit doesn't support all exchanges including the Indian Exchanges. This is why I refer to American exchanges. For example the company WIPRO (Wipro Limited) is listed on the American Exchanges under WIT so if I would use that ticker I'd find the relevant data again. I need to build in a proper error handling here, I'll put it on the to-do.

You can find this company by searching by name:

```
import financedatabase as fd

equities = fd.Equities()

equities.search(name='wipro', exclude_exchanges=True)
```

| symbol   | name | currency   | sector                 | industry_group      | industry    | exchange   | market                  | country   |   state | city      |   zipcode | website              | market_cap   |   isin |   cusip |   figi |   composite_figi |   shareclass_figi |
|:---------|:--------------|:-----------|:-----------------------|:--------------------|:------------|:-----------|:------------------------|:----------|--------:|:----------|----------:|:---------------------|:-------------|-------:|--------:|-------:|-----------------:|------------------:|
| WIT      | Wipro Limited | USD        | Information Technology | Software & Services | IT Services | NYQ        | New York Stock Exchange | India     |     nan | Bengaluru |    560035 | http://www.wipro.com | Large Cap    |    nan |     nan |    nan |              nan |               nan |

This returns the data you are looking for:

```
from financetoolkit import Toolkit

wipro = Toolkit('WIT', api_key='FMP_KEY')

wipro.get_balance_sheet_statement()
```

|                                          |         2019 |         2020 |         2021 |          2022 |          2023 |
|:-----------------------------------------|-------------:|-------------:|-------------:|--------------:|--------------:|
| Cash and Cash Equivalents                | 158529000000 | 144499000000 | 169793000000 |  103836000000 |   91880000000 |
| Short Term Investments                   | 230099000000 | 190685000000 | 176326000000 |  247445000000 |  318328000000 |
| Cash and Short Term Investments          | 388628000000 | 335184000000 | 346119000000 |  351281000000 |  410208000000 |
| Accounts Receivable                      | 150858000000 | 164255000000 | 153875000000 |  215066000000 |  214957000000 |
| Inventory                                |   3951000000 |   1865000000 |   1064000000 |    1334000000 |    1188000000 |
| Other Current Assets                     |  28469000000 |    135000000 |    783000000 |    1223000000 |   32899000000 |
| Total Current Assets                     | 571906000000 | 519851000000 | 523186000000 |  620752000000 |  661096000000 |
| Property, Plant and Equipment            |  70601000000 |  97868000000 | 101612000000 |  109768000000 |  107361000000 |
| Goodwill                                 | 116980000000 | 131012000000 | 139127000000 |  246989000000 |  307970000000 |
| Intangible Assets                        |  20085000000 |  20897000000 |  13085000000 |   43555000000 |   43045000000 |
| Long Term Investments                    |   8928000000 |  11487000000 |  12368000000 |   20309000000 |   27830000000 |
| Tax Assets                               |   5604000000 |   6005000000 |   1664000000 |    2298000000 |    2100000000 |
| Other Fixed Assets                       |  39067000000 |  29942000000 |  40392000000 |   35511000000 |   26420000000 |
| Fixed Assets                             | 261265000000 | 297211000000 | 308248000000 |  458430000000 |  514726000000 |
| Other Assets                             |            0 |            0 |            0 |             0 |             0 |
| Total Assets                             | 833171000000 | 817062000000 | 831434000000 | 1079182000000 | 1175822000000 |
| Accounts Payable                         |  28527000000 |  78129000000 |  78870000000 |   99034000000 |   89054000000 |
| Short Term Debt                          |  71099000000 |  79762000000 |  83543000000 |  104289000000 |   97441000000 |
| Tax Payables                             |   9541000000 |  11731000000 |  17324000000 |   13231000000 |   18846000000 |
| Deferred Revenue                         |  26129000000 |  20239000000 |  23393000000 |   30126000000 |   22682000000 |
| Other Current Liabilities                |  79054000000 |  38263000000 |  44234000000 |   74880000000 |   58576000000 |
| Total Current Liabilities                | 214350000000 | 216393000000 | 230040000000 |  308329000000 |  267753000000 |
| Long Term Debt                           |  28368000000 |  17478000000 |  20971000000 |   71640000000 |   77225000000 |
| Deferred Revenue Non Current             |            0 |  17112000000 |    123000000 |             0 |             0 |
| Deferred Tax Liabilities                 |   3417000000 |   2825000000 |   4633000000 |   12141000000 |   15153000000 |
| Other Non Current Liabilities            |  16283000000 |  21033000000 |  21074000000 |   28399000000 |   33938000000 |
| Total Non Current Liabilities            |  48068000000 |  41336000000 |  46801000000 |  112180000000 |  126316000000 |
| Other Liabilities                        |            0 |            0 |            0 |             0 |             0 |
| Capital Lease Obligations                |            0 |  19198000000 |  21182000000 |   24233000000 |   24573000000 |
| Total Liabilities                        | 262418000000 | 257729000000 | 276841000000 |  420509000000 |  394069000000 |
| Preferred Stock                          |            0 |            0 |            0 |             0 |             0 |
| Common Stock                             |  12068000000 |  11427000000 |  10958000000 |   10964000000 |   10976000000 |
| Retained Earnings                        | 534700000000 | 519907000000 | 466692000000 |  551252000000 |  660964000000 |
| Accumulated Other Comprehensive Income   |            0 |            0 |  41154000000 |   47061000000 |   46803000000 |
| Other Total Shareholder Equity           |  21348000000 |  26124000000 |  34291000000 |   48881000000 |   62421000000 |
| Total Shareholder Equity                 | 568116000000 | 557458000000 | 553095000000 |  658158000000 |  781164000000 |
| Total Equity                             | 568116000000 | 557458000000 | 553095000000 |  658158000000 |  781164000000 |
| Total Liabilities and Shareholder Equity | 830534000000 | 817062000000 | 831434000000 | 1079182000000 | 1175822000000 |
| Minority Interest                        |            0 |   1875000000 |   1498000000 |     515000000 |     589000000 |
| Total Liabilities and Equity             | 830534000000 | 817062000000 | 831434000000 | 1079182000000 | 1175822000000 |
| Total Investments                        | 239027000000 | 202172000000 | 188694000000 |  267754000000 |  346158000000 |
| Total Debt                               |  99467000000 |  97240000000 | 104514000000 |  175929000000 |  174666000000 |
| Net Debt                                 | -59062000000 | -47259000000 | -65279000000 |   72093000000 |   82786000000 |

**Reply by poonkundran-k** [reply]
Yes, thank you. It seems that for other companies financial statement (other than ones listed in US or ADR), we have to pay them the fees.

**Reply by ptyson17** [reply]
hi Jeroen, is this just an issue for India or does it apply to Australian companies as well?

Further, is this just an issue with FMP? If we get the paid API would Australian (.AX) companies come through on these functions?

**Reply by JerBouma** [reply]
In general, symbols that include e.g. `.AX` will not work for the majority of providers. What these symbols represent is that the same company can be listed on multiple exchanges. When you would do the following:

```
equities = fd.Equities()
equities.search(country='Australia', exclude_exchanges=True)
```

You will obtain more than 500 Australian companies listed on the US exchanges with symbols that will work with the FinanceToolkit (and thus FMP). As an example, 'BHP' works well with the FinanceToolkit (being BHP Group) and you can also see on all the other exhanges it is listed with `equities.search(name='BHP', exclude_exchanges=False)` which returns:

| symbol    | name                            | currency   | sector                 | industry_group      | industry        | exchange   | market                                | country       | state   | city      |   zipcode | website                   | market_cap   | isin         |   cusip | figi         | composite_figi   | shareclass_figi   |
|:----------|:--------------------------------|:-----------|:-----------------------|:--------------------|:----------------|:-----------|:--------------------------------------|:--------------|:--------|:----------|----------:|:--------------------------|:-------------|:-------------|--------:|:-------------|:-----------------|:------------------|
| 0HN3.L    | BHP Group                       | USD        | Materials              | Materials           | Metals & Mining | LSE        | London Stock Exchange (international) | Australia     | VIC     | Melbourne |      3000 | http://www.bhp.com        | Large Cap    | AU000000BHP4 |     nan | nan          | nan              | nan               |
| B1BL34.SA | BHP Group                       | BRL        | Materials              | Materials           | Metals & Mining | SAO        | Bovespa Soma                          | Australia     | VIC     | Melbourne |      3000 | http://www.bhp.com        | Large Cap    | AU000000BHP4 |     nan | nan          | nan              | nan               |
| BBL       | BHP Group                       | USD        | Materials              | Materials           | Metals & Mining | NYQ        | New York Stock Exchange               | Australia     | VIC     | Melbourne |      3000 | http://www.bhp.com        | Large Cap    | AU000000BHP4 |     nan | nan          | nan              | nan               |
| BHP       | BHP Group                       | USD        | Materials              | Materials           | Metals & Mining | NYQ        | New York Stock Exchange               | Australia     | VIC     | Melbourne |      3000 | http://www.bhp.com        | Large Cap    | AU000000BHP4 |     nan | BBG000BD5SC8 | BBG000BD5NZ4     | BBG001S5P5K4      |
| BHP.AX    | BHP Group                       | AUD        | Materials              | Materials           | Metals & Mining | ASX        | Australian Securities Exchange        | Australia     | VIC     | Melbourne |      3000 | http://www.bhp.com        | Large Cap    | AU000000BHP4 |     nan | nan          | nan              | nan               |
| BHP.BA    | BHP Group                       | ARS        | Materials              | Materials           | Metals & Mining | BUE        | Buenos Aires Mercato De Valores       | Australia     | VIC     | Melbourne |      3000 | http://www.bhp.com        | Large Cap    | AU000000BHP4 |     nan | nan          | nan              | nan               |
| BHP.BE    | BHP Group                       | EUR        | Materials              | Materials           | Metals & Mining | BER        | Berlin Stock Exchange                 | Australia     | VIC     | Melbourne |      3000 | http://www.bhp.com        | Large Cap    | AU000000BHP4 |     nan | BBG000C5G4L5 | BBG000C5G104     | BBG001S5P5K4      |
| BHP.DU    | BHP Group                       | EUR        | Materials              | Materials           | Metals & Mining | DUS        | Dusseldorf Stock Exchange             | Australia     | VIC     | Melbourne |      3000 | http://www.bhp.com        | Large Cap    | AU000000BHP4 |     nan | BBG000C5G1Y7 | BBG000C5G104     | BBG001S5P5K4      |
| BHP.F     | BHP Group                       | EUR        | Materials              | Materials           | Metals & Mining | FRA        | Frankfurt Stock Exchange              | Australia     | VIC     | Melbourne |      3000 | http://www.bhp.com        | Large Cap    | AU000000BHP4 |     nan | BBG000C5G1K2 | BBG000C5G104     | BBG001S5P5K4      |
| BHP.JO    | BHP Group                       | ZAc        | Materials              | Materials           | Metals & Mining | JNB        | Johannesburg Stock Exchange           | Australia     | VIC     | Melbourne |      3000 | http://www.bhp.com        | Large Cap    | AU000000BHP4 |     nan | nan          | nan              | nan               |
| BHP.L     | BHP Group                       | GBP        | Materials              | Materials           | Metals & Mining | LSE        | London Stock Exchange (international) | Australia     | VIC     | Melbourne |      3000 | http://www.bhp.com        | Large Cap    | AU000000BHP4 |     nan | nan          | nan              | nan               |
| BHP.MU    | BHP Group                       | EUR        | Materials              | Materials           | Metals & Mining | MUN        | Munich Stock Exchange                 | Australia     | VIC     | Melbourne |      3000 | http://www.bhp.com        | Large Cap    | AU000000BHP4 |     nan | BBG000C5G3Z2 | BBG000C5G104     | BBG001S5P5K4      |
| BHP.SG    | BHP Group                       | EUR        | Materials              | Materials           | Metals & Mining | STU        | Stuttgart Stock Exchange              | Australia     | VIC     | Melbourne |      3000 | http://www.bhp.com        | Large Cap    | AU000000BHP4 |     nan | BBG000C5G2P5 | BBG000C5G104     | BBG001S5P5K4      |
| BHP1.BE   | BHP Group                       | EUR        | Materials              | Materials           | Metals & Mining | BER        | Berlin Stock Exchange                 | Australia     | VIC     | Melbourne |      3000 | http://www.bhp.com        | Large Cap    | AU000000BHP4 |     nan | BBG000F2J1N0 | BBG000F2HXZ9     | BBG001S5YBT2      |
| BHP1.DU   | BHP Group                       | EUR        | Materials              | Materials           | Metals & Mining | DUS        | Dusseldorf Stock Exchange             | Australia     | VIC     | Melbourne |      3000 | http://www.bhp.com        | Large Cap    | AU000000BHP4 |     nan | BBG000F2HYW0 | BBG000F2HXZ9     | BBG001S5YBT2      |
| BHP1.F    | BHP Group                       | EUR        | Materials              | Materials           | Metals & Mining | FRA        | Frankfurt Stock Exchange              | Australia     | VIC     | Melbourne |      3000 | http://www.bhp.com        | Large Cap    | AU000000BHP4 |     nan | BBG000F2HY96 | BBG000F2HXZ9     | BBG001S5YBT2      |
| BHP1.HA   | BHP Group                       | EUR        | Materials              | Materials           | Metals & Mining | HAN        | Hanover Stock Exchange                | Australia     | VIC     | Melbourne |      3000 | http://www.bhp.com        | Large Cap    | AU000000BHP4 |     nan | nan          | nan              | nan               |
| BHP1.HM   | BHP Group                       | EUR        | Materials              | Materials           | Metals & Mining | HAM        | Hamburg Stock Exchange                | Australia     | VIC     | Melbourne |      3000 | http://www.bhp.com        | Large Cap    | AU000000BHP4 |     nan | BBG000F2J3H3 | BBG000F2HXZ9     | BBG001S5YBT2      |
| BHP1.MU   | BHP Group                       | EUR        | Materials              | Materials           | Metals & Mining | MUN        | Munich Stock Exchange                 | Australia     | VIC     | Melbourne |      3000 | http://www.bhp.com        | Large Cap    | AU000000BHP4 |     nan | BBG000F2J150 | BBG000F2HXZ9     | BBG001S5YBT2      |
| BHP1.SG   | BHP Group                       | EUR        | Materials              | Materials           | Metals & Mining | STU        | Stuttgart Stock Exchange              | Australia     | VIC     | Melbourne |      3000 | http://www.bhp.com        | Large Cap    | AU000000BHP4 |     nan | BBG000F2J0N2 | BBG000F2HXZ9     | BBG001S5YBT2      |
| BHPA      | BHPA Inc.                       | USD        | Information Technology | Software & Services | Software        | PNK        | OTC Bulletin Board                    | United States | FL      | Miami     |     33131 | http://www.bhpadvance.com | Nano Cap     | nan          |     nan | nan          | nan              | nan               |
| BHPBF     | BHP Group                       | USD        | Materials              | Materials           | Metals & Mining | PNK        | OTC Bulletin Board                    | Australia     | VIC     | Melbourne |      3000 | http://www.bhp.com        | Large Cap    | AU000000BHP4 |     nan | nan          | nan              | nan               |
| BHPG34.SA | BHP Group                       | BRL        | Materials              | Materials           | Metals & Mining | SAO        | Bovespa Soma                          | Australia     | VIC     | Melbourne |      3000 | http://www.bhp.com        | Large Cap    | AU000000BHP4 |     nan | nan          | nan              | nan               |
| BHPLF     | BHP Group                       | USD        | Materials              | Materials           | Metals & Mining | PNK        | OTC Bulletin Board                    | Australia     | VIC     | Melbourne |      3000 | http://www.bhp.com        | Large Cap    | AU000000BHP4 |     nan | nan          | nan              | nan               |
| BHPN.MX   | BHP Group                       | MXN        | Materials              | Materials           | Metals & Mining | MEX        | Mexico Stock Exchange                 | Australia     | VIC     | Melbourne |      3000 | http://www.bhp.com        | Large Cap    | AU000000BHP4 |     nan | BBG000L47GH6 | BBG000L47GH6     | BBG001S5P5K4      |
| BIL.BE    | BHP Group                       | EUR        | Materials              | Materials           | Metals & Mining | BER        | Berlin Stock Exchange                 | Australia     | VIC     | Melbourne |      3000 | http://www.bhp.com        | Large Cap    | AU000000BHP4 |     nan | nan          | nan              | nan               |
| BIL.DE    | BHP Group                       | EUR        | Materials              | Materials           | Metals & Mining | GER        | XETRA                                 | Australia     | VIC     | Melbourne |      3000 | http://www.bhp.com        | Large Cap    | AU000000BHP4 |     nan | nan          | nan              | nan               |
| BIL.DU    | BHP Group                       | EUR        | Materials              | Materials           | Metals & Mining | DUS        | Dusseldorf Stock Exchange             | Australia     | VIC     | Melbourne |      3000 | http://www.bhp.com        | Large Cap    | AU000000BHP4 |     nan | nan          | nan              | nan               |
| BIL.F     | BHP Group                       | EUR        | Materials              | Materials           | Metals & Mining | FRA        | Frankfurt Stock Exchange              | Australia     | VIC     | Melbourne |      3000 | http://www.bhp.com        | Large Cap    | AU000000BHP4 |     nan | nan          | nan              | nan               |
| BIL.HA    | BHP Group                       | EUR        | Materials              | Materials           | Metals & Mining | HAN        | Hanover Stock Exchange                | Australia     | VIC     | Melbourne |      3000 | http://www.bhp.com        | Large Cap    | AU000000BHP4 |     nan | nan          | nan              | nan               |
| BIL.HM    | BHP Group                       | EUR        | Materials              | Materials           | Metals & Mining | HAM        | Hamburg Stock Exchange                | Australia     | VIC     | Melbourne |      3000 | http://www.bhp.com        | Large Cap    | AU000000BHP4 |     nan | nan          | nan              | nan               |
| BIL.MU    | BHP Group                       | EUR        | Materials              | Materials           | Metals & Mining | MUN        | Munich Stock Exchange                 | Australia     | VIC     | Melbourne |      3000 | http://www.bhp.com        | Large Cap    | AU000000BHP4 |     nan | nan          | nan              | nan               |
| BIL.SG    | BHP Group                       | EUR        | Materials              | Materials           | Metals & Mining | STU        | Stuttgart Stock Exchange              | Australia     | VIC     | Melbourne |      3000 | http://www.bhp.com        | Large Cap    | AU000000BHP4 |     nan | nan          | nan              | nan               |
| BIL1.BE   | BHP Group                       | EUR        | Materials              | Materials           | Metals & Mining | BER        | Berlin Stock Exchange                 | Australia     | VIC     | Melbourne |      3000 | http://www.bhp.com        | Large Cap    | AU000000BHP4 |     nan | nan          | nan              | nan               |
| BIL1.F    | BHP Group                       | EUR        | Materials              | Materials           | Metals & Mining | FRA        | Frankfurt Stock Exchange              | Australia     | VIC     | Melbourne |      3000 | http://www.bhp.com        | Large Cap    | AU000000BHP4 |     nan | nan          | nan              | nan               |
| BIL1.SG   | BHP Group                       | EUR        | Materials              | Materials           | Metals & Mining | STU        | Stuttgart Stock Exchange              | Australia     | VIC     | Melbourne |      3000 | http://www.bhp.com        | Large Cap    | AU000000BHP4 |     nan | nan          | nan              | nan               |
| C77010.TI | EFG ANGLOAM/BHP/RIOTINT EXP 010 | nan        | nan                    | nan                 | nan             | TLO        | EuroTLX                               | nan           | nan     | nan       |       nan | nan                       | nan          | nan          |     nan | nan          | nan              | nan               |

**Reply by ptyson17** [reply]
thanks Jeroen. In the end, I did subscribe to the paid version of FMP which brings through information from ASX companies. I actually have a separate question about an error I discovered with ratios - will create a separate discussion items for that.

**Reply by JerBouma** [reply]
> thanks Jeroen. In the end, I did subscribe to the paid version of FMP which brings through information from ASX companies. I actually have a separate question about an error I discovered with ratios - will create a separate discussion items for that.

Great, can you do this within the FinanceToolkit repository instead of FinanceDatabase? See: https://github.com/JerBouma/FinanceToolkit
